Sources/SwiftFormat/PrettyPrint/PrettyPrint.swift

```diff
@@ -686,15 +699,28 @@ public class PrettyPrinter {
       case .commaDelimitedRegionStart:
         lengths.append(0)
 
-      case .commaDelimitedRegionEnd(_, let isSingleElement):
+      case .commaDelimitedRegionEnd(let isCollection, _, let isSingleElement):
         // The token's length is only necessary when a comma will be printed, but it's impossible to
         // know at this point whether the region-start token will be on the same line as this token.
         // Without adding this length to the total, it would be possible for this comma to be
         // printed in column `maxLineLength`. Unfortunately, this can cause breaks to fire
         // unnecessarily when the enclosed tokens comma would fit within `maxLineLength`.
-        let length = isSingleElement ? 0 : 1
-        total += length
-        lengths.append(length)
+        let shouldHandleCommaDelimitedRegion: Bool? =
+          switch configuration.trailingCommasInMultilineLists {
+          case .always:
+            true
+          case .never:
+            false
+          case .ignore:
+            isCollection ? configuration.multiElementCollectionTrailingCommas : nil
+          }
+        if let shouldHandleCommaDelimitedRegion, shouldHandleCommaDelimitedRegion {
```

Comment:
This could just be `if shouldHandleCommaDelimitedRegion == true`. Being written twice made me have to reread it a couple times to make sure they were referring to the same thing.